.github/workflows/full_kubeflow_integration_test.yaml

```diff
@@ -139,8 +148,15 @@ jobs:
         sleep 30
 
     - name: Run Training Operator Test
+      run: ./tests/gh-actions/test_training_operator.sh "${KF_PROFILE}"
+
+    - name: Run KServe Test
       run: |
-        sed 's/namespace: .*/namespace: '"$KF_PROFILE"'/g' tests/gh-actions/kf-objects/training_operator_job.yaml | kubectl apply -f -
+        sed "s/PROFILE_NAMESPACE/$KF_PROFILE/g" tests/gh-actions/kf-objects/kserve_test.yaml | kubectl apply -f -
+        sleep 30
```

Comment:
I think we can do better than sleep here in a follow up PR. Inferenceservices have a readiness field.